Web/starter_project_barbies/components/team/TeamCard.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { BsFacebook, BsLinkedin, BsInstagram } from 'react-icons/bs'
+import Image from "next/image";
+
+type TeamCardProps = {
+  name: string
+  jobTitle: string
+  description: string
+  image: string
+  facebook: string
+  linkedin: string
+  instagram: string
+}
+
+const TeamCard = ({
+  name,
+  jobTitle,
+  description,
+  image,
+  facebook,
+  linkedin,
+  instagram,
+}: TeamCardProps) => {
+  return (
+    <div className="team-card box-content font-sans w-72 mt-4 text-center shadow-md shadow-gray-200 rounded-md py-2 m-auto">
+      <div className="team-card__image container overflow-hidden my-4">
+        <Image className="rounded-full mx-auto w-32" width={200} height={200} src={image} alt={name} />
+      </div>
+
+      <div className="team-card__content m-4">
```

Comment:
i don't think you need to prefix everything with team-card
you can just define the team card scss file and only import it in this component

Reply:
actually idk where these classes are being used 
are they only for semantics?

Reply:
No, they are not getting used now. It was in case we need to add custom css later on.

Reply:
well if you are not using it i say get rid of it